platform/src/api/v1/surveys/mod.rs

```diff
@@ -30,88 +30,42 @@ pub enum Status {
 
 #[server(endpoint = "/v1/surveys/lists", input = GetUrl, output = Json)]
 pub async fn list_surveys(
+    email: String,
     _size: Option<i32>,
     _bookmark: Option<String>,
     _status: Option<Status>,
 ) -> Result<CommonQueryResponse<SurveySummary>, ServerFnError> {
     dioxus_logger::tracing::debug!("/v1/surveys/lists",);
 
+    use crate::api::aws::ses::AuthKeyModel;
+    use easy_dynamodb::error::DynamoException;
+
+    let log = crate::utils::logger::new_api("GET", &format!("/v1/surveys/lists"));
+    let cli = crate::utils::db::get(&log);
+
+    let survey_models: Result<(Option<Vec<SurveySummary>>, Option<String>), DynamoException> = cli
+        .find("gsi1-index", None, Some(100), vec![("gsi1", email)])
+        .await;
+
+    let surveys = match survey_models {
+        Ok(v) => v.0.unwrap(),
+        Err(_e) => {
+            vec![]
+        }
+    };
+
+    let mut surveys_res: Vec<SurveySummary> = vec![];
+
+    for survey in surveys {
+        surveys_res.push(SurveySummary {
+            created_at: survey.created_at / 1000,
+            updated_at: survey.updated_at / 1000,
+            ..survey
+        });
+    }
+
     Ok(CommonQueryResponse {
```

Comment:
이렇게 하면 위에 라인 거의다 지울수 있을거에요. 우리 dagit 의 collection 조회 API 가 어떻게 구현되어 있는지 참고하시면 좋을듯 합니다. 

```suggestion
    Ok(CommonQueryResponse::query(log, "gsi1-index", bookmark, size, vec![("gsi1", email)])
```

Reply:
아 네 이렇게 하니깐 더 간결하게 코드가 작성될 수 있네요!
수정했습니다!